paddle/platform/gpu_info.h

```diff
@@ -42,6 +43,18 @@ size_t GpuMinChunkSize();
 //! Get the maximum chunk size for GPU buddy allocator.
 size_t GpuMaxChunkSize();
 
+//! Copy memory from address src to dst asynchronously.
+void GpuMemcpyAsync(void *dst, const void *src, size_t count,
```

Comment:
Should we move these copying functions into a new source file, say, `copy.{h,cc}`? I am not sure. Just mention it.